components/selection/selection-input.js

```diff
@@ -40,6 +40,8 @@ class Input extends SkeletonMixin(LabelledMixin(LitElement)) {
 			 * @type {string}
 			 */
 			key: { type: String },
+			/* passthrough for list-item */
+			disabledTooltip: { state: true },
```

Comment:
I tried making this `_disabledTooltip` since it does't work on `selection-single` so I didn't expose it publicly, but linting complained when I did `._disabledTooltip=${...}`.

Reply:
I think I'd prefer just add a linting exception and mark it private.